tools/test/h5dump/testh5dump.sh.in

```diff
@@ -1435,8 +1495,12 @@ TOOLTEST3 non_existing.ddl --enable-error-stack tgroup.h5 non_existing.h5
 # test to verify HDFFV-9407: long double full precision
 GREPTEST OUTTXT "1.123456789012345" t128bit_float.ddl -m %.35Lf t128bit_float.h5
 
+# test HDFFFV-10480, using corrupted file from a CVE group, before the fix,
+# h5dump would segfault.
+TOOLTEST_FAIL t10480H5O_fill.ddl t10480H5O_fill.h5
+
 # Clean up temporary files/directories
-CLEAN_TESTFILES_AND_TESTDIR
+#CLEAN_TESTFILES_AND_TESTDIR
```

Comment:
Did you mean to leave this commented?

Reply:
> Did you mean to leave this commented?

Of course not! :-)  However, I realized I accidentally combined the new commit with an existing PR again!  Is there anyway for me to retract so they can be in two separate PRs?

Reply:
Of course there is.  One possibility (of many): 
- close this PR
- create a new fork 
- create 2 new branches from hdf5_1_10
- copy changes for each PR to one of the branches
- create the PRs
- sooner or later delete one of the forks

Reply:
Haha... I was hoping for not that!! ;-)  I guess that's the most straightforward for me.  Thank you, Larry!
Oh, wait, that means I need to delete bmribler/hdf5?  I will lose all the related stuff, right?